Content.Shared/Network/NetMessages/MsgUpdateAdminStatus.cs

```diff
@@ -60,7 +62,7 @@ public override void WriteToBuffer(NetOutgoingMessage buffer)
 
             if (isAdmin)
             {
-                buffer.Write(Admin.Active);
+                buffer.Write(Admin!.Active);
```

Comment:
Removed `new()` initialization as I'm pretty sure it would mess stuff up, then changed the code so the compiler understands that Admin can't be null after the if.